__tests__/utils.tsx

```diff
@@ -115,7 +114,7 @@ describe('getPathInfo()', () => {
     serverMock(
```

Comment:
on line 105-111 I get
expect(received).toEqual(expected) // deep equality

    Expected: {"currentPath": "/current-path", "typename": "Article"}

as error

Reply:
The error was, that calls to the API are POST requests and no GET requests. `serverMock` mocks GET requests and thus these tests fail. I have implemented now to mock functions: `mockHttpGet()` and `mockHttpPost()` in order to distinguish between both cases.